corehq/util/workbook_reading/tests/test_spreadsheets.py

```diff
@@ -40,27 +40,53 @@ def _assert_worksheets_equal(self, worksheet1, worksheet2):
 @run_on_all_adapters(SpreadsheetCellTypeTest)
 def test_xlsx_types(self, open_workbook, ext):
     with open_workbook(get_file('types', ext)) as workbook:
-        self.assert_workbooks_equal(
-            workbook,
-            Workbook(
-                worksheets=[
-                    make_worksheet(title='Sheet1', rows=[
-                        ['String', 'Danny'],
-                        ['Date', date(1988, 7, 7)],
-                        ['Date Time', datetime(2016, 1, 1, 12, 0)],
-                        ['Time', time(12, 0)],
-                        ['Midnight', time(0, 0)],
-                        ['Int', 28],
-                        ['Int.0', 5],
-                        ['Float', 5.1],
-                        ['Bool-F', False],
-                        ['Bool-T', True],
-                        ['Empty', None],
-                        ['Percent', 0.49],
-                        ['Calculation', 2],
-                        ['Styled', 'Styled'],
-                        ['Empty Date', None],
-                    ]),
-                ]
+        if ext == 'xlsx':
+            self.assert_workbooks_equal(
+                workbook,
+                Workbook(
+                    worksheets=[
+                        make_worksheet(title='Sheet1', rows=[
+                            ['String', 'Danny'],
+                            ['Date', date(1988, 7, 7)],
+                            ['Date Time', datetime(2016, 1, 1, 12, 0)],
+                            ['Time', time(12, 0)],
+                            ['Midnight', date(1899, 12, 30)],
```

Comment:
This is strange.  Adding this commit to run remainder of tests, but I want to understand what's going on here before merging.

Reply:
https://bitbucket.org/openpyxl/openpyxl/issues/936/datetime-1899-12-30-0-0-is-returned-when-0

Reply:
Here is the test that had to be fixed:
https://travis-ci.org/dimagi/commcare-hq/jobs/470126342

Reply:
@dannyroberts Any chance you remember how you picked these test cases?  It looks like with the openpyxl upgrade there is a slight difference in how this row is handled:
<img width="141" alt="screen shot 2018-12-19 at 3 23 29 pm" src="https://user-images.githubusercontent.com/1757035/50246009-15fd0700-03a2-11e9-9c72-e9a0416adcdd.png">